merkle/map_verifier_test.go

```diff
@@ -98,182 +69,34 @@ var mapInclusionTestVector = []struct {
 			testonly.MustDecodeBase64("CtC2GCsc3/zFn1DNkoUThUnn7k+DMotaNXvmceKIL4Y="),
 		},
 		testonly.MustDecodeBase64("U6ANU1en3BSbbnWqhV2nTGtQ+scBlaZf9kRPEEDZsHM="),
-	}, {
-		"key-1-848",
```

Comment:
We're not losing any coverage.  The cases weren't being used, and I don't think they were exercising any additional code paths.  

In the next PR, I'll be adding a fuller suite of merkle tree tests that don't rely on test vectors, and can thus be used with any hasher.